src/tests/account/test_secp256k1.cairo

```diff
@@ -54,18 +53,17 @@ fn test_unpack_big_secp256k1_points() {
 
     // Check point 2
 
-    let (expected_x, expected_y) = big_point_2.get_coordinates().unwrap();
+    let (expected_x, _) = big_point_2.get_coordinates().unwrap();
 
     let (xlow, xhigh_and_parity) = StorePacking::pack(big_point_2);
-    let (x, y) = StorePacking::unpack((xlow, xhigh_and_parity)).get_coordinates().unwrap();
+    let (x, _) = StorePacking::unpack((xlow, xhigh_and_parity)).get_coordinates().unwrap();
 
     assert_eq!(x, expected_x);
 }
 
 #[test]
 fn test_secp256k1_serialization() {
     let (big_point_1, big_point_2) = get_points();
-    let curve_size = Secp256k1Impl::get_curve_size();
```

Comment:
huh

Reply:
2.5 issues warnings for unused variables

Reply:
yeah, i was a bit more surprised of the unused line, which i believe made it through the audit